Twig/Extension/CKEditorExtension.php

```diff
@@ -15,20 +15,22 @@
 
 /**
  * @author Marodon Jérémy <marodon.jeremy@gmail.com>
+ *
+ * @deprecated to remove in SonataFormatterBundle 3.x version, which will drop EgeloenCKEditorBundle 2.x and 3.x support
  */
 class CKEditorExtension extends \Twig_Extension
 {
     /**
-     * @var CKEditorHelper
+     * @var CKEditorHelper|CKEditorRendererInterface
      */
     private $helper;
 
     /**
      * Creates a CKEditor extension bridge for BC.
      *
-     * @param CKEditorHelper $helper The CKEditor helper.
+     * @param CKEditorHelper|CKEditorRendererInterface $helper The CKEditor helper or renderer.
      */
-    public function __construct(CKEditorHelper $helper)
+    public function __construct($helper)
```

Comment:
I think this would be BC break if this class have children. Let's do it for 2.4. Have to make a lot of class `final` @greg0ire...